src/LanguageServer/Impl/Completion/ImportCompletion.cs

```diff
@@ -60,7 +59,7 @@ internal class ImportCompletion {
 
         public static CompletionResult GetCompletionsInFromImport(FromImportStatement fromImport, CompletionContext context) {
             // No more completions after '*', ever!
-            if (fromImport.Names != null && fromImport.Names.Any(n => n?.Name == "*" && context.Position > n.EndIndex)) {
+            if (fromImport.Names.Any(n => n?.Name == "*" && context.Position > n.EndIndex)) {
```

Comment:
Before, `Names` was checked for null, but now it isn't. Does this need something like `?.Any(...) ?? false)` or `MaybeEnumerate().Any()`?

Reply:
I now see that you only removed the null check, so maybe it's known not-null.

Reply:
It is `ImmutableArray`, VS shows it as unnecessary check although I don't mind.

Reply:
Ah, yeah. It's probably upgrading the ImmutableArray to ImmutableArray? and checking null, which is pointless.

Reply:
It's a struct `ImmutableArray` so VS shows it as unnecessary check although I don't mind either way.